spacehammer/lib/testing/test.lua

```diff
@@ -1,15 +1,21 @@
 --test.lua
 -- A script to run fennel files as tests passed in as cli args
 
+local scriptPath = _cli.args[2] .. "/"
 
-fennel = require("spacehammer.vendor.fennel")
+if _cli.args[3] == nil then
+  print("Need to specify tests")
+  return
+end
 
--- Support docstrings
+package.path = package.path .. ";" .. scriptPath .. "?.lua;" .. scriptPath .. "?/init.lua;"
+package.cpath = package.cpath .. ";" .. scriptPath .. "?.so;"
 
-local searcher = fennel.makeSearcher({
```

Comment:
this was never used?

Reply:
I think you're right. I've ran into that situation a few times when bootstrapping fennel where I can't `(require export :some.path.to.a.fennel.file)` then exhaust all options before remembering I need to change fennel.path 😅 Should be fine without this bit.